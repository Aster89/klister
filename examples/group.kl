#lang kernel
(import (shift kernel 1))

(define a 1)
(meta
  (define a 10))
(group
  (define b a)
  (define c b)
  (meta
    (define b a)
    (define c b)))

(example a)
(example b)
(example c)
(meta
  (example a)
  (example b)
  (example c))
