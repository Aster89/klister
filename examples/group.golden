Signal 1 : Signal
Signal 1 : Signal
Signal 1 : Signal
Signal 10 : Signal
Signal 10 : Signal
Signal 10 : Signal
